apps/ejabberd/src/gen_mod.erl

```diff
@@ -229,7 +229,6 @@ wait_for_stop1(MonitorReference) ->
 get_opt(Opt, Opts) ->
     case lists:keysearch(Opt, 1, Opts) of
         false ->
-            % TODO: replace with more appropriate function
```

Comment:
@ludwikbukowski, any comment on that?

Reply:
throwing exception here if fine for me.
Hence "TODO" can be removed